### PR TITLE
[Rule] Classic Invite Requires Target

### PR DESCRIPTION
**Pull request by fryguy503**
Default is false, when enabled, group invites will require a hard target to invite.

`/invite charname` will no longer function when enabled.

### CODE REVIEW

File: zone/client_packet.cpp [hunk 1]
```diff
@@ -7321,23 +7321,25 @@ void Client::Handle_OP_GroupInvite2(const EQApplicationPacket *app)
 	}
 
 	GroupInvite_Struct* gis = (GroupInvite_Struct*)app->pBuffer;
+	
+	Mob* Invitee = nullptr;
```
Comment (Kinglykrab) [inline zone/client_packet.cpp:7325]: `invitee`?

File: zone/client_packet.cpp [hunk 2]
```diff
@@ -7346,39 +7348,36 @@ void Client::Handle_OP_GroupInvite2(const EQApplicationPacket *app)
...
-	}
-	else
-	{
-		auto pack = new ServerPacket(ServerOP_GroupInvite, sizeof(GroupInvite_Struct));
-		memcpy(pack->pBuffer, gis, sizeof(GroupInvite_Struct));
-		worldserver.SendPacket(pack);
-		safe_delete(pack);
+	} else {
+		if (RuleB(Character, GroupInvitesRequireTarget)) {
+			Message(Chat::White, "Invalid target!");
```
Comment (Kinglykrab) [inline zone/client_packet.cpp:7374]: Is this the message Live sent, or just one you put here for some sort of feedback?

Comment (fryguy503) [reply zone/client_packet.cpp:7374]: Feedback Only, nothing similar in eqstr_us that i could locate

Comment (joligario) [reply zone/client_packet.cpp:7374]: `12268 You must target a player first to invite to join your group.`

Comment (fryguy503) [reply zone/client_packet.cpp:7374]: Thanks, no idea how I missed that one

Comment (fryguy503) [reply zone/client_packet.cpp:7374]: 12268 You must target a player or use /invite <name> to invite someone to your group.

That wont work for this case.

Comment (joligario) [reply zone/client_packet.cpp:7392]: Wasn't for the string code, was more for the text feedback

Comment (fryguy503) [reply zone/client_packet.cpp:7392]: Oh, I got you!

**Review by Kinglykrab — APPROVED** [review]

**Pull request merged**